include/swift/AST/ASTContext.h

```diff
@@ -580,6 +580,13 @@ class ASTContext {
   /// Does nothing in non-asserts (NDEBUG) builds.
   void verifyAllLoadedModules() const;
 
+  /// \brief Check whether the module with a given name can be imported without
+  /// importing it.
+  ///
+  /// Note that if errors occur during the module loading process this check may
```

Comment:
That's not *quite* the condition we're going for. It's more "even if this check succeeds, errors may still occur if the module is eventually loaded", right?